src/lib.rs

```diff
@@ -259,15 +289,15 @@ impl Handler for CorsHandlerAllowAny {
         match req.headers.get::<headers::Origin>() {
             None => {
                 self.handler.handle(req)
-            },
+            }
```

Comment:
Oh, I didn't know that these commas are optional. Is this the recommended style?

Reply:
I 've just run https://github.com/rust-lang-nursery/rustfmt

I assume it follows the [RFC](https://github.com/rust-lang/rfcs/pull/1607)

Reply:
Ok, that's fine. :)